docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,57 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Examples
+
+### Download Files in IE
+
+TestCafe prevents native browser dialogs, including those before file download. In Internet Explorer, however, you should disable this dialog in browser settings, because it cannot be dismissed programmatically.
+
+The following example shows how to download the file once you disable the dialog:
+
+```html
+<body>
+    <form method="get" action="./src/file.zip">
+        <button id="downloadButton" type="submit">Download!</button>
+     </form>
+</body>
+```
+
+This sample page includes a button that downloads a file from the `/src` folder on the server. To obtain the file, use a [RequestHook](../../reference/test-api/requesthook/README.md):
+
+```js
+import fs from 'fs';
+import { Selector, RequestHook } from 'testcafe';
+
+class MyHook extends RequestHook {
+    constructor (requestFilterRules, responseEventConfigureOpts) {
+        super(requestFilterRules, responseEventConfigureOpts);
+    }
+
+    onRequest (event) {
+    }
+
+    onResponse (event) {
+        const path = 'downloadedFile.zip';
+
+        fs.writeFileSync(path, event.body);
+   }
+}
+
+fixture `fixture`
+    .page `https://localhost:5500/downloadPage.html`;
+
+test.requestHooks(new MyHook({url: /.+\/src\/.+/, method: 'GET' } , {
+    includeHeaders: true,
+    includeBody:    true
+}))
+
+(`test`, async t => {  
+    await t.click(Selector('#downloadButton'));
+
+    await t.wait(10000);
```

Comment:
Add comment here

// Wait for file download.

// See the recommended way how to wait file download in https://github.com/DevExpress/testcafe-examples/tree/master/examples/wait-for-file-download
await t.wait(10000);

Reply:
added